src/Components/NavBar.css

```diff
@@ -0,0 +1,41 @@
+.NavBar {
+    background-color: #000080;
+    padding: 20px;
+    position: sticky;
+    
+}
+
+.NavBar .NavLinks {
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.NavBar .NavLinks a{
+    color: white;
+    text-decoration: none;
+}
+
+.NavBar .NavLinks .siteName {
+    color: rgb(72, 206, 72);
+    font-weight: bold;
+    font-size: large;
+    
+}
+
+.NavBar .NavLinks .actionButtons {
+   display: flex;
+   justify-content: space-between;
```

Comment:
This would mean that all of these become flex-box. I think you don't intend that